HTML/js/rejestracja.js

```diff
@@ -0,0 +1,149 @@
+window.onload = function(e){
+    console.log('onload')
+    }
+    
+    function validatereg() {
+
+        let errors = true;
+        console.log(errors)
+        if (!emailvalidation('email')) {
+            errors = false;
+        }
+        if (!requiredvalidation('lastname')) {
+            errors = false;
+        }
+        if (!requiredvalidation('firstname')) {
+            errors = false;
+        }
+        if (!requiredvalidation('username')) {
+            errors = false;
+        }
+
+        if(!birthdatevalidation('birthdate')) {
+            errors = false;
+        }
+
+        if (!checkpassword('password')) {
+            errors = false;
+        }
+
+        if (!errors ){
+            document.getElementById("error").setAttribute("style", "display: inline;")
+        } else {
+            document.getElementById("error").setAttribute("style", "display: none;")
+        }
+    }
+
+    function emailvalidation(element){
+        let email = document.getElementById(element)
+        let error = false;
+        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+        if(!email.value){
+            document.getElementById('errors_'+element).innerHTML = "Pole jest wymagane"
+            error = true;
+        } else if(!re.test(String(email.value).toLowerCase())) {
+            document.getElementById('errors_'+element).innerHTML = "błędny email"
+            error = true;
+        } 
+        
+        if(error === true) {
+            showError(email);
+            return false;
+        } else {
+            document.getElementById('errors_'+element).innerHTML = ""
+            hideError(email)
+            return true
+        }
+    }
+    
+    
+    function requiredvalidation(element) {
+        let name = document.getElementById(element)
```

Comment:
Expected indentation of 2 spaces but found 8.
'name' is never reassigned. Use 'const' instead.
Missing semicolon.